src/index.html

```diff
@@ -9,6 +9,26 @@
     <meta name="apple-mobile-web-app-capable" content="yes"/>
     <meta name="apple-mobile-web-app-status-bar-style" content="black">
     <link rel="shortcut icon" type="image/png" ng-href="{{favicon}}"/>
+    <link rel="apple-touch-icon" sizes="57x57" href="{{touchicons}}/apple-touch-icon-57x57.png">
+    <link rel="apple-touch-icon" sizes="60x60" href="{{touchicons}}/apple-touch-icon-60x60.png">
+    <link rel="apple-touch-icon" sizes="72x72" href="{{touchicons}}/apple-touch-icon-72x72.png">
+    <link rel="apple-touch-icon" sizes="76x76" href="{{touchicons}}/apple-touch-icon-76x76.png">
+    <link rel="apple-touch-icon" sizes="114x114" href="{{touchicons}}/apple-touch-icon-114x114.png">
+    <link rel="apple-touch-icon" sizes="120x120" href="{{touchicons}}/apple-touch-icon-120x120.png">
+    <link rel="apple-touch-icon" sizes="144x144" href="{{touchicons}}/apple-touch-icon-144x144.png">
+    <link rel="apple-touch-icon" sizes="152x152" href="{{touchicons}}/apple-touch-icon-152x152.png">
+    <link rel="apple-touch-icon" sizes="180x180" href="{{touchicons}}/apple-touch-icon-180x180.png">
+    <link rel="icon" type="image/png" href="{{touchicons}}/favicon-32x32.png" sizes="32x32">
+    <link rel="icon" type="image/png" href="{{touchicons}}/favicon-194x194.png" sizes="194x194">
+    <link rel="icon" type="image/png" href="{{touchicons}}/favicon-96x96.png" sizes="96x96">
+    <link rel="icon" type="image/png" href="{{touchicons}}/android-chrome-192x192.png" sizes="192x192">
+    <link rel="icon" type="image/png" href="{{touchicons}}/favicon-16x16.png" sizes="16x16">
+    <link rel="manifest" href="{{touchicons}}/manifest.json">
```

Comment:
use this path
<link rel="manifest" href="{{themeFolder}}/manifest.json">

manifest.json il related to the whole theme and not to icons. Move the manifest in the theme folder at first level please.